g2core/board/ArduinoDue/gShield-pinout.h

```diff
@@ -258,12 +258,12 @@ pin_number kADC14_PinNumber = -1;  // Not physially pinned out
 
 
 // GRBL / gShield compatibility pins -- Due board ONLY
-
+// NOTE: These dont appear to work
```

Comment:
Well I assumed `INPUT_ACTION_CYCLE_START` would work exactly like a `~` but bypassing serial, same with `INPUT_ACTION_HOLD` (but it doesnt work as intended), anyways i will mess around with these and try to make them work in my branch.